src/portal/src/app/base/project/project-detail/project-detail.component.html

```
@@ -1,7 +1,9 @@
-<span class="back-icon"><</span>
-<a *ngIf="hasSignedIn" (click)="backToProject()" class="backStyle">{{
-    'PROJECT_DETAIL.PROJECTS' | translate
-}}</a>
+<ng-container *ngIf="hasSignedIn || isPublicAndNotLogged">
```

HTMLHint (1.9.2)

[error] Doctype must be declared before any non-comment content.

(doctype-first)